### PR TITLE
#65 use debug library instead of logger.debug

### PR DESCRIPTION
**Pull request by lojzatran**
Fixes: https://github.com/commercetools/commercetools-node-variant-reassignment/issues/65

### CODE REVIEW

**Comment by lojzatran** [comment]
@junajan https://github.com/commercetools/commercetools-node-variant-reassignment/pull/67/commits/2ae597e54ec7809752d672ceb5089f87fc1a26f8

File: lib/services/transaction-manager.js [hunk 1]
```diff
@@ -1,19 +1,19 @@
 import Promise from 'bluebird'
 import * as constants from '../constants'
 
+import pkg from '../../package.json'
```
Comment (junajan) [inline lib/services/transaction-manager.js]: Include logger also her instead of building new one

Comment (lojzatran) [reply lib/services/transaction-manager.js]: https://github.com/commercetools/commercetools-node-variant-reassignment/pull/67/commits/839dfb2441ac25df97cd48dce6f5c23a4bed81d4

**Comment by lojzatran** [comment]
@butenkor I will do the documentation in a different PR. I added your comments here: https://github.com/commercetools/commercetools-node-variant-reassignment/issues/17

**Pull request merged**